Source/WebKit/NetworkProcess/storage/IDBStorageManager.cpp

```diff
@@ -339,7 +339,7 @@ void IDBStorageManager::unregisterTransaction(WebCore::IDBServer::UniqueIDBDatab
 std::unique_ptr<WebCore::IDBServer::IDBBackingStore> IDBStorageManager::createBackingStore(const WebCore::IDBDatabaseIdentifier& identifier)
 {
     if (m_path.isEmpty() || identifier.isTransient())
-        return makeUnique<WebCore::IDBServer::MemoryIDBBackingStore>(identifier);
+        return makeUnique<WebCore::IDBServer::SQLiteMemoryIDBBackingStore>(identifier);
```

Comment:
It would be nice if we had a generic SQLite `create` function that took an `identifier` and returned a `SQLiteMemoryIDBackingStore` if it was transient, and a `SQLIteIDBackingStore` if it was not. But I guess we need the file path in the latter case, so the create function would be different anyway,